### ISSUE TITLE
Clicking on menu item closes submenu (Link)

### ISSUE THREAD

**Issue by SusanBrown** [Thaerious/NERVE#28]

**Comment by Thaerious** [comment]
Is this still occurring, I can't seem to replicate it on the mac.

**Comment by SusanBrown** [comment]
Yes, this is happening consistently with the right menu panel in a mac, testing duncsa-short. Clicking on Tag Selection causes the submenu to disappear.

**Comment by abramoffcwrc** [comment]
Happened for me today on PC on Chrome and Firefox.
